TTHAnalysis/python/plotter/susy-ewkino/3l/mca_ewkino.txt

```diff
@@ -29,60 +40,117 @@ dummy : TTWToLNu : 0 ; FakeRate="susy-ewkino/3l/fakerate_standard_central.txt",
-#_standard_prompt_WG               : WGToLNuG                        : xsec             ; FillColor=ROOT.kOrange+7, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isconv.txt", Label="WG"
-#_standard_prompt_ZG               : ZGTo2LG                         : xsec             ; FillColor=ROOT.kOrange+7, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isconv.txt", Label="ZG"
-_standard_prompt_XG               : TTGJets                         : xsec             ; FillColor=ROOT.kOrange+7, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isconv.txt", Label="X+G (conv.)"
-_standard_prompt_XG               : TGJets                          : xsec             ; FillColor=ROOT.kOrange+7, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isconv.txt", Label="X+G (conv.)"
-_standard_prompt_XG               : WGToLNuG                        : xsec             ; FillColor=ROOT.kOrange+7, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isconv.txt", Label="X+G (conv.)"
-_standard_prompt_XG               : ZGTo2LG                         : xsec             ; FillColor=ROOT.kOrange+7, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isconv.txt", Label="X+G (conv.)"
-_standard_prompt_Rares            : DYJetsToLL_M10to50              : 3.039/9.225*xsec ; FillColor=ROOT.kViolet-6, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isprompt.txt", Label="Rares (prompt)"
-_standard_prompt_Rares            : DYJetsToLL_M10to50_ext1         : 6.186/9.225*xsec ; FillColor=ROOT.kViolet-6, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isprompt.txt", Label="Rares (prompt)"
-_standard_prompt_Rares            : DYJetsToLL_M50                  : xsec             ; FillColor=ROOT.kViolet-6, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isprompt.txt", Label="Rares (prompt)"
-_standard_prompt_Rares            : TTTT                            : 250./1210.*xsec  ; FillColor=ROOT.kViolet-6, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isprompt.txt", Label="Rares (prompt)"
-_standard_prompt_Rares            : TTTT_ext                        : 960./1210.*xsec  ; FillColor=ROOT.kViolet-6, FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isprompt.txt", Label="Rares (prompt)"
+_standard_prompt_WZ               : WZTo3LNu                        : xsec             ; FillColor=ROOT.kOrange  , FakeRate="susy-ewkino/3l/fakerate_standard_central.txt\,susy-ewkino/3l/fakerate_load_isprompt.txt", Label="WZ (prompt)"
```

Comment:
please replace all these calls to fakerate files to apply simple cuts with cuts written in the mca:
_standard_prompt_WZ : WZTo3LNu : xsec : (Lep1_isPrompt && ....) ; FillColor=ROOT.kOrange